### PR TITLE
Update generateLoginPwd.py

### PR DESCRIPTION
**Pull request by ErnandesAJr**
Another variable was created to name the files with the default <tenant: device>, because when making a request to Dojot via "/sign/<cname>/pcks10" you should only use the device id in cname and no <tenant: device>. Also, it is noteworthy that you should change the ":" for another character like "-", because if you keep the ":" the script will only work on Linux systems and not on Windows.

### CODE REVIEW

**Comment by mprevide** [comment]
Thanks for the contribution. 
For change to -, maybe we need to update : to - in ejbca-rest and iot-agent-mosca too. I will check before merge your PR.

**Comment by ErnandesAJr** [comment]
There is no need to change the ejbca-rest and iot-agent-mosca. I updated the Readme to a version I have been using for some time.

**Comment by mprevide** [comment]
> There is no need to change the ejbca-rest and iot-agent-mosca. I updated the Readme to a version I have been using for some time.

Sorry, I got your changes wrong ...
Please check  the conflicts in the readme .

Thx

**Comment by mprevide** [comment]
There are tutorial show using the certificate retriever with mqtt:

For dojot version > 0.5 
https://dojotdocs.readthedocs.io/en/latest/mqtt-tls.html#id19

For old versions
https://dojotdocs.readthedocs.io/en/v0.4.3/mosca-tls.html

File: README.md [hunk 1]
```diff
@@ -1,2 +1,95 @@
...
+# View the generated certificates
+# Enter certs folder
+computer@name:~/certificate-retriver$ cd certs
+# List what is inside the folder
+computer@name:~/certificate-retriver/certs$ ls # Or dir if windows
+IOTmidCA.crt  admin-339e11.crt  admin-339e11.csr  admin-339e11.key
+computer@name:~/certificate-retriver/certs$
+```
+
+### Example of how to publish with `mosquitto_pub` with the certificates
```
Comment (mprevide) [inline README.md:80]: Example of how to publish with `mosquitto_pub` with the certificates on IoT Agent MQTT Mosca

**Comment by mprevide** [comment]
These changes should go to the branch release/v0.4.4, because in development the script is a little different, considers the tenant in cname.

**Comment by luissajunior** [comment]
Não mais compatível com a versão corrente da Dojot, por isso ele será fechado.

**Pull request closed without merging**